src/main/kotlin/no/nav/dagpenger/inntekt/db/PostgresInntektStore.kt

```diff
@@ -23,7 +23,7 @@ class PostgresInntektStore(private val dataSource: DataSource) : InntektStore {
             return using(sessionOf(dataSource)) { session ->
                 session.run(
                         queryOf(
-                                """SELECT inntektId
+                                """SELECT DISTINCT inntektId
```

Comment:
note sure this will work, as we inntektId is not serial to postgresql and cannot sort by itself what is "newest". 
i'm pretty sure we have to `order by` e.g.: 

```
SELECT DISTINCT ON (inntektId)
inntektId
FROM inntekt_V1_arena_mapping
ORDER BY inntektId, timestamp DESC;
```

Reply:
Add a test for this? when fixed the constraint probelm